### ISSUE TITLE
Question: Can mouse enter/leave be detected for a specific item (selected item) in UWP ListBox?

### ISSUE THREAD

**Issue by YoungFcode** [microsoft/microsoft-ui-xaml#3976]
I'm using a UWP ListBox and would like to be able to detect when the mouse pointer enters / leaves the SelectedItem in ListBox list.
The ListBox does have an associated PointerExited event, but that is for the whole ListBox itself. 
I need to detect the mouse pointer on the specific item.

**Comment by mdtauk** [comment]
Have you tried looking at the ListBoxItem?

**Comment by YoungFcode** [comment]
Do you mean putting ControlTemplate for ListBoxItem inside ListBox? 
Then yes I tried this, but no event fired:
                        <ControlTemplate TargetType="ListBoxItem">
                                <Border
                                    PointerExited="ListBoxItem_PointerExited">
                                </Border>
                        </ControlTemplate>

**Comment by YoungFcode** [comment]
Is there a ListBoxItem 'PointerExited' Visual State?

**Comment by mdtauk** [comment]
> Is there a ListBoxItem 'PointerExited' Visual State?

Could you check if the ListBoxItem moves from the **PointerOver** state back to the **Default/Rest** state?

Looking at the docs, all UIElements have a **PointerExited** event
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.uielement.pointerexited?view=winrt-19041

**Comment by YoungFcode** [comment]
Indeed, the UIElements do have a PointerExited event. Where would I find an example of the PointerExited event implementation for a UWP ListBoxItem?

**Comment by StephenLPeters** [comment]
ListBox is a pretty old control, I'd recomend using ListView instead if possible.  However attaching to the PointerExited event of your ListBoxItem should be pretty easy. I don't have an example on hand but if you have retrieved the ListBoxItem as a local variable it should be as easy as:

`LocalListBoxItem.PointerExited += MyPointerExitedHandler`

We have examples where we do this for other controls in the repo: https://github.com/microsoft/microsoft-ui-xaml/blob/2c6f1994bbae555a459902a5cd083f9e1a97c45f/dev/PipsPager/TestUI/PipsPagerPage.xaml.cs#L52

If you are having trouble getting the item yourself you might need this https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.itemscontrol.containerfromitem?view=winrt-19041

What is your scenario? This shouldn't really be needed.
